core/messagerouter.go

```diff
@@ -0,0 +1,10 @@
+package core
+
+import (
+	"github.com/insolar/insolar/messagerouter"
+)
```

Comment:
Vanya, this can not go into master. core should not import anything. My branch has proper solution